package.json

```diff
@@ -106,6 +105,7 @@
     "@babel/preset-typescript": "^7.14.5",
     "@babel/runtime": "^7.14.6",
     "@babel/types": "^7.14.5",
+    "@craco/craco": "^6.4.3",
```

Comment:
Doesn't officially support CRA v5 but if the CI passes then I got it to work. v7 is under development and will add support for v5.